home/templates/email/email_rsvp.html

```diff
@@ -0,0 +1,24 @@
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>RVSP Confirmation</title>
+    </head>
+    <body>
+        <div class="container">
+            Hello {{ user.first_name }},
```

Comment:
I think we need to use `<p>` tags here or `<br />`. I don't think the line breaks and spacing will be respected by an email client, but I could be wrong. I'm about 60% sure. The txt email will respect the `\n` characters from the file, so that one should be good.